core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1348,83 +1313,64 @@ public Api getApi() {
     }
 
     /**
-     * The file we save our configuration.
+     * Loads a {@link Fingerprint} from the Storage with the given unique id.
+     * @return Loaded {@link Fingerprint}. Null if the config file does not exist or
+     * malformed.
      */
-    private static @NonNull XmlFile getConfigFile(@NonNull File file) {
-        return new XmlFile(XSTREAM,file);
+    public static @CheckForNull Fingerprint load(@NonNull String id) throws IOException {
+        long start=0;
```

Comment:
please stick to the same format: 
start = 0;

Reply:
@StefanSpieker Thanks for pointing it out :)